tools/proofers/my_projects.php

```diff
@@ -52,12 +51,12 @@
     $pool_sort_options
 );
 
-// Update saved view and sort settings
-$userSettings->set_value("my_projects:round_view", $round_view);
-$userSettings->set_value("my_projects:pool_view", $pool_view);
-$userSettings->set_value("my_projects:round_sort", $round_sort);
-$userSettings->set_value("my_projects:pool_sort", $pool_sort);
-
+// Update saved view and sort settings if they've changed
+foreach (["round_view", "pool_view", "round_sort", "pool_sort"] as $setting) {
+    if ($$setting != $userSettings->get_value("my_projects:$setting")) {
```

Comment:
is getting the current value expensive or does not writing make up for this cost?

Reply:
Getting the current value is an in-memory operation from within the `$userSettings` object so it's very cheap.